controllers/provisioning_enabled.go

```diff
@@ -36,18 +36,14 @@ func EnabledFeatures(ctx context.Context, osClient osclientset.Interface) (v1alp
 		return features, errors.Wrap(err, "unable to determine Platform")
 	}
 
-	if infra.Status.ControlPlaneTopology == osconfigv1.ExternalTopologyMode {
-		return features, nil
-	}
-
 	//NOTE(dtantsur/janders) if the code below is updated, GetServerInternalIP() function in provisioning/utils.go needs to be updated to match.
 	switch infra.Status.PlatformStatus.Type {
-	case osconfigv1.BareMetalPlatformType, osconfigv1.NonePlatformType:
+	case osconfigv1.BareMetalPlatformType, osconfigv1.NonePlatformType, osconfigv1.KubevirtPlatformType:
```

Comment:
Let's maybe delay this change? It enables iPXE provisioning from KubeVirt, which will require a bunch of additional testing and documenting. I think they should be fine with virtual media only for now.

Reply:
Done